src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -106,6 +107,7 @@ void TargetedMovementGeneratorMedium<T, D>::_setTargetLocation(T* owner, bool up
         && owner->HasUnitState(UNIT_STATE_FOLLOW));
 
     bool result = i_path->CalculatePath(x, y, z, forceDest);
+    i_path->ReducePathLenghtByDist(owner->GetObjectSize() + i_target->GetObjectSize() + CONTACT_DISTANCE, i_target->ToUnit()); // this distance is correct only for MoveChase with no offset
```

Comment:
ReducePathLenghtByDist also valide the last point of the path and if no right point is found in range and in los of the target, the path is set to flagged as unusable. This is why. (regarding the spelling, i didn't create the method. i just rewrote it, but yeah i will fix that)